apps/sentry-client-desktop/src/features/keys/HasKeys.tsx

```diff
@@ -245,7 +245,7 @@ export function HasKeys({combinedOwners, combinedLicensesMap, statusMap, isWalle
 							}
 							setSelectedValue={setSelectedWallet}
 							getDropdownItems={getDropdownItems}
-							extraClasses={{dropdown: "max-w-[456px]"}}
+							extraClasses={{dropdown: "max-w-[456px]", dropdownOptions: "w-[538px]"}}
```

Comment:
Same as above, I have not checked the appearance of the dropdown in the desktop app, but I'd assume these widths should be adjusted down to match the existing widths, but I did not want to change without a review.

Reply:
@XDapps is width `538px` correct here, seems to be `456px` ?

Is there anything I can assis you with running / testing the desktop app locally ?

Reply:
It was using the 538px value before(hardcoded) so I didn't think I should change since it was not part of this story, but I can update now.

No, I was able to get the app to run locally. I checked those widths were being used but they were also overflowing. 

I have fixed both now.